types/time.go

```diff
@@ -263,7 +264,7 @@ func (t Time) ToNumber() *MyDecimal {
 
 	s, err := t.DateFormat(tfStr)
 	if err != nil {
-		log.Error("Fatal: never happen because we've control the format!")
+		logutil.Logger(context.Background()).Error("[fatal] never happen because we've control the format!")
```

Comment:
```suggestion
		logutil.Logger(context.Background()).Error("[error] never happen because we've control the format!")
```
Do we need this prefix? If needed, should we use fatal?

Reply:
I think it's OK. The log level is `error`, we needn't log it again.